spec/presenters/hyrax/collection_presenter_spec.rb

```diff
@@ -126,12 +122,6 @@
     it { is_expected.to eq ['adc12v'] }
   end
 
-  describe "#size", :clean_repo do
```

Comment:
Should we tweak these tests to verify that the hard-coded value is returned? And/or that `Deprecation` receives `:warn`?